ci/build/upload_build_info.sh

```diff
@@ -6,10 +6,14 @@ set -ex
 readonly PIPELINE_POSTMERGE="0189e759-8c96-4302-b6b5-b4274406bf89"
 readonly PIPELINE_CIV1_BRANCH="0183465b-c6fb-479b-8577-4cfd743b545d"
 if [[
-  "${BUILDKITE_PIPELINE_ID:-}" != "${PIPELINE_POSTMERGE}" && 
-  "${BUILDKITE_PIPELINE_ID:-}" != "${PIPELINE_CIV1_BRANCH}" &&
-  "${BUILDKITE_BRANCH:-}" != "master"
+    "${BUILDKITE_PIPELINE_ID:-}" != "${PIPELINE_POSTMERGE}" && 
+    "${BUILDKITE_PIPELINE_ID:-}" != "${PIPELINE_CIV1_BRANCH}" 
 ]]; then
+  echo "Skip build info uploading on postmerge."
+  exit 0
+fi
+
+if [[ "${BUILDKITE_BRANCH:-}" != "master"]]; then
   echo "Skip build info uploading on master branch."
```

Comment:
nit: I think you mean skip uploading on "non-master" branch?

Reply:
w00t